python_modules/dagster/dagster/core/storage/runs/sql_run_storage.py

```diff
@@ -767,6 +784,8 @@ def wipe(self):
             conn.execute(RunTagsTable.delete())  # pylint: disable=no-value-for-parameter
             conn.execute(SnapshotsTable.delete())  # pylint: disable=no-value-for-parameter
             conn.execute(DaemonHeartbeatsTable.delete())  # pylint: disable=no-value-for-parameter
+            if self.snapshot_cache:
+                self.snapshot_cache.clear_all()
```

Comment:
nit but I feel like we're kind of gating on `snapshot_cache`'s existence everywhere